cloud/spawn.go

```diff
@@ -69,6 +72,9 @@ func (so *SpawnOptions) validate() error {
 		return err
 	}
 
+	if !evergreen.UseSpawnHostRegions && so.Region != "" && so.Region != evergreen.DefaultEC2Region {
```

Comment:
I don't totally follow how the feature flag works. This creates an error if `so.Region != ""`, for example, but is this just if a spawnhost is configured with a region, and we don't want it to be. It seems more like a thing to do a sanity check instead of an actual feature flag, since it doesn't really enable behavior. Is that your intent?

Reply:
There's going to be a significant period of time where we've created flags or options that the user shouldn't be able to see or use yet, because build hasn't set up regions. I want to encapsulate those things using a global that we can flip later (but a different solution is fine too).

Here, we create an error if the user has defined the region as anything but the default, because we aren't supporting anything but the default.

Reply:
This can potentially check instead that the region is defined in admin settings, but I will probably want a global eventually to hide UI things or etc.